spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+RSpec.describe Airbrake::Shoryuken::ErrorHandler do
+  let(:error) { AirbrakeTestError.new('shoryuken error') }
+  let(:body) { { message: 'message' } }
+  let(:notice) { double 'notice' }
+  let(:worker) { FooWorker.new }
+  let(:endpoint) do
+    'https://airbrake.io/api/v3/projects/113743/notices?key=fd04e13d806a90f96614ad8e529b2822'
+  end
+
+  class FooWorker; end
```

Comment:
@kyrylo I don't find it unusual, actually my first attempt was with `Class.new`, the issue with that is because we use [`worker.class.to_s`](https://github.com/phstc/airbrake/blob/add-shoryuken-support/lib/airbrake/shoryuken/error_handler.rb#L21), which will return something like `Class:0x007fd03e57c360`.

Ideally, I think this should use [`worker_name`](https://github.com/phstc/shoryuken/blob/master/lib/shoryuken/util.rb#L40), but I didn't want to add Shoryuken as a development dependency in the gemspec.

WDYT?

> To avoid defining a class in global scope

If we go back with unwrapping the name space it won't be that global anymore.